modules/kubernetes-addons/cilium/variables.tf

```diff
@@ -0,0 +1,26 @@
+variable "helm_config" {
+  description = "Helm Config for cilium"
+  type        = any
+  default     = {}
+}
+
+variable "manage_via_gitops" {
```

Comment:
make sure to add the relevant argo proj file to the GitOps addons repo too

Reply:
https://github.com/aws-samples/eks-blueprints-add-ons/pull/74